tests/test_surface/test_regular_surface_vs_grd3d.py

```diff
@@ -68,6 +68,13 @@ def test_get_surface_from_grd3d_porosity(tmpdir, generate_plot):
     assert surfr.values.mean() == pytest.approx(0.1667, abs=0.01)
 
 
+def test_loadvalues_raises_error():
+    grd = xtgeo.grid3d.Grid(RGRD2, fformat="roff")
+    srf = xtgeo.surface_from_grid3d(grd)
+    with pytest.raises(ValueError):
```

Comment:
Could you add a `match=` parameter to the raise check?

Reply:
👍